<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgCalligraphyToolOptions</class>
 <widget class="QWidget" name="WdgCalligraphyToolOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="gbProfiles">
     <property name="title">
      <string>Tool Presets</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cmbProfiles"/>
      </item>
      <item>
       <widget class="QToolButton" name="bnSaveProfile">
        <property name="toolTip">
         <string>Save profile as...</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="document-save-as">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bnRemoveProfile">
        <property name="toolTip">
         <string>Remove profile</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KisDoubleSliderSpinBox" name="sldCaps" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="gbStrokeConfig">
     <property name="title">
      <string>Stroke configuration</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="KisPopupButton" name="bnSize">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KisPopupButton" name="bnRotation">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KisPopupButton" name="bnRatio">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbGB_Adjust">
     <property name="title">
      <string>Adjust Position</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rdNoAdjust">
        <property name="text">
         <string>No adjustment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdAdjustPath">
        <property name="text">
         <string>Follow Selected Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdAdjustAssistant">
        <property name="text">
         <string>Follow Assistants</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSampling">
     <property name="title">
      <string>Sampling Interval</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="KisDoubleSliderSpinBox" name="sldTimeInterval" native="true">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The Time Interval samples based on the time it takes to make the stroke. The value is in milliseconds.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KisDoubleSliderSpinBox" name="sldDistanceInterval" native="true">
        <property name="toolTip">
         <string>The Distance interval samples based on the length of the stroke in the coordinates of the screen. So if you zoom in, you can make preciser strokes despite having the same distance interval.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisPopupButton</class>
   <extends>QPushButton</extends>
   <header>kis_popup_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
